src/main/java/org/jenkinsci/test/acceptance/po/ArtifactArchiver.java

```diff
@@ -15,7 +19,8 @@ public ArtifactArchiver includes(String value) {
     }
 
     public ArtifactArchiver excludes(String value) {
-        control("advanced-button").click();
+        // TODO fix form-element-path plugin for new button markup
```

Comment:
some buttons (e.g. yui ones) no longer have a path set after buttons rework,

Guessing it's around here:
https://github.com/jenkinsci/form-element-path-plugin/blob/master/src/main/resources/org/jenkinsci/plugins/form-element-path.js#L103-L106

Can be reproduced with `core.FreestyleJobTest#archiveArtifacts` if anyone wants to try fix it, if its high impact I'll take a look, I tried in the browser debugger briefly but it just kept getting call constantly even when I wasn't clicking anything so hard to iterate on it.

cc @janfaracik

Reply:
Hmm, there's 176 tests failing, first one I looked at was because of this issue.

I think next step will be to fix above and then see what we're looking at

Reply:
https://github.com/jenkinsci/form-element-path-plugin/pull/11

I'll revert this change and re-run a suite once it's released and then we can see where we are at

Reply:
resolved by moving to core and a couple of tweaks in the JavaScript